vaadin-button-flow-parent/vaadin-button-flow/src/test/java/com/vaadin/flow/component/button/tests/ButtonTest.java

```diff
@@ -314,12 +314,6 @@ public void disableOnClick_click_componentIsDisabled() {
         Assert.assertFalse(
                 "Button should have been disabled when event has been fired",
                 buttonIsEnabled.get());
-
-        StateNode node = button.getElement().getNode();
-        HashMap<String, Serializable> changeTracker = node.getChangeTracker(
-                node.getFeature(ElementAttributeMap.class), () -> null);
-        Assert.assertEquals("Disabled attribute should be set to true", "true",
-                changeTracker.get("disabled"));
```

Comment:
This was testing implementation details of the old workaround that don't apply anymore. I think it's safe to remove with having integration tests in place.